.npmignore

```diff
@@ -1,6 +1,17 @@
 # exclude development artifacts
 test
 node_modules
-bower_components
 .editorconfig
-.eslintrcn
+.eslintrc
+.eslintignore
+.idea
+webapp
+
+# only for testing, these resources should not be delivered but are easy to forget
+webapp/resources
+webapp/test-resources
+dist/webapp/resources
```

Comment:
src and dist shall be included in npm module, but not test

Reply:
dist contains all data that is part of webapp (src). By including it in npm module we have it duplicate